src/main/java/com/minenash/monsters_in_the_closet/mixin/BedBlockMixin.java

```diff
@@ -29,7 +29,7 @@ public class BedBlockMixin {
     @Unique
     private static BlockPos blockPos = null;
     
-    @Inject(method = "method_19283",
+    @Inject(method = "method_19283(Lnet/minecraft/entity/player/PlayerEntity;Lnet/minecraft/entity/player/PlayerEntity$SleepFailureReason;)V",
```

Comment:
This is not correct (mixin is not injected), but I can't find how to inject into that lamba correctly atm.